src/components/UserInfo.js

```diff
@@ -2,21 +2,26 @@ class UserInfo {
   constructor({ nameSelector, infoSelector, imageSelector }) {
     this._nameElement = document.querySelector(nameSelector);
     this._infoElement = document.querySelector(infoSelector);
-    //  this._imageElement = document.querySelector(imageSelector);
+    this._avatarElement = document.querySelector(imageSelector);
   }
 
   getUserInfo() {
     return {
       title: this._nameElement.textContent,
       description: this._infoElement.textContent,
-      // image: this._imageElement.src
     };
   }
 
-  setUserInfo({ title, description, image }) {
-    this._nameElement.textContent = title;
-    this._infoElement.textContent = description;
-    // this._imageElement.src = image
+  setUserInfo({ name, about, avatar }) {
+    this._nameElement.textContent = name;
+    this._infoElement.textContent = about;
+    this._avatarElement.src = avatar;
```

Comment:
Please, reuse the `setUserAvatar` method here to avoid code duplication


```js


  setUserInfo({ name, about, avatar }) {
    this._nameElement.textContent = name;
    this._infoElement.textContent = about;
    this.setUserAvatar(avatar);
  }

  setUserAvatar(avatar) {
    this._avatarElement.src = avatar;
  }

```